packages/date-picker/src/vaadin-date-picker.js

```diff
@@ -163,7 +163,8 @@ class DatePicker extends DatePickerMixin(InputControlMixin(ThemableMixin(Element
         theme$="[[__getOverlayTheme(theme, _overlayInitialized)]]"
         on-vaadin-overlay-open="_onOverlayOpened"
         on-vaadin-overlay-close="_onOverlayClosed"
-        on-vaadin-overlay-outside-click="focus"
+        restore-focus-on-close
```

Comment:
**nit:** I wonder if it would be reasonable to move changes unrelated to combo-box to a separate PR and mark it as `refactor:`?

It might make it easier to find out in which commit the date-picker has been changed in case some bugs will come out of these changes in the future. Not a big deal though.

Reply:
Sounds reasonable. 👍